Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackStateMachineEventsTests.swift

```diff
@@ -72,14 +73,41 @@ class AVFoundationPlaybackStateMachineEventsTests: QuickSpec {
                     }
                 }
 
-                context("when play and seek to end") {
+//                context("when play and seek to end") {
```

Comment:
Will we leave this block of code commented? We want this?

Reply:
we made a few tests with the stalling event adjustment (like bruno explained last tech session). And since we slitted the PR, we had to comment "correct tests" which are failing. We'll uncomment those when the other PR comes.
But if you want to, we can just delete it and then rewrite it on stalling event adjustment PR. Which do you prefer?

Reply:
I did not know that. We can leave it like this then. ;)